public/audiograph.js

```diff
@@ -181,6 +183,65 @@ class ADSRNode extends AudioNode
     }
 }
 
+/**
+ * BitCrush distortion
+ * See https://github.com/maximecb/noisecraft/issues/81
+ */
+export class BitCrush extends AudioNode
+{
+    constructor(id, state, sampleRate, send)
+    {
+        super(id, state, sampleRate, send);
+        this.configure(state.params.bitdepth, state.params.factor, state.params.useAvg);
+    }
+
+    configure(numBits, factor, useAvg) {
+        console.log(`BitCrush node: ${this.nodeId}, numBits: ${numBits}, factor: ${factor}, useAvg: ${useAvg}`);
```

Comment:
You should derive the `setParam` and `setState` methods and make sure that they call `configure` so that the node will change its behavior when a parameter in the UI is changed.